better_image_click.py

```diff
@@ -48,7 +46,7 @@ def click_image_rand(  # pylint: disable=too-many-arguments
     return True
 
 
-def mouse_button(delay=0.1, button="left") -> None:
+def mouse_button(delay: float = 0.1, button: str = "left") -> None:
```

Comment:
I believe these are considered superfluous type definitions in Python since the types are already known by the initial assignment -- a few linters all warned me against doing that in the first place.

This applies to a good chunk of these, so I won't re-post this multiple times.